codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/generators/EnumGenerator.kt

```diff
@@ -299,7 +309,7 @@ private fun RustWriter.renderForwardCompatibilityNote(
         variant in a current version of SDK, your code should continue to work when you
         upgrade SDK to a future version in which the enum does include a variant for that
         feature.
-        """.trimIndent(),
+        """,
```

Comment:
Interesting that `trimIdent` can be removed 🤔  I had to add it only because a pre-commit hook complained about it.

Reply:
I think they can always be removed, and perhaps the pre-commit hook error message is not clear/just wrong.